tool/src/webapp/js/clog.js

```diff
@@ -1,5 +1,3 @@
-import {loadProperties} from "/webcomponents/sakai-i18n.js";
```

Comment:
Should this be removed? Is it no longer used?

Reply:
This was removed by you here:
https://github.com/sakaiproject/sakai/commit/f5902019164ca2719cdc03645b424913a541c589#diff-f05975a9c3b3bc9f87864797fe83d85c54664761ac7f6af098ebac4079276eadL1

Reply:
Ah, right. It's in the base.js bundle now which is globally loaded. I remember now. Cheers.

Reply:
There are still other problems, but with these changes at least the tool is loaded.

![image](https://github.com/user-attachments/assets/67d043b9-7768-4faf-b3e8-ba5dc63a3838)

Reply:
Please merge. Thanks!